### ISSUE TITLE
Missing vocalized confirmation when PLP rearranges results on page

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#15217]
**Describe the bug**
When facets, sort by or display grid are triggered, the results on the page will rearrange without any audible confirmation that the page view has changed.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to PLP using a screen reader and keyboard
2. Trigger any of the following: facets, sort by, display grid button.
3. Hear the silence.

**Expected behaviour**
For any of the aforementioned options being triggered a message will be vocalized: _"Page view updated with your selected options"_.

<img width="1301" alt="Screen Shot 2022-02-22 at 10 09 23 AM" src="https://user-images.githubusercontent.com/15113219/155194781-66e74dcd-678f-4a8d-b5b0-bfceb40c53d5.png">

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.

vocalized message when 
- selected facets
<img width="1020" alt="Screen Shot 2022-03-10 at 2 36 26 PM" src="https://user-images.githubusercontent.com/45409515/157741203-22975d02-f96a-4652-9db5-642804681ce0.png">

![PLP10](https://user-images.githubusercontent.com/45409515/157741217-b4c41014-2acf-4118-92d1-a03a6b8ab954.png)

![PLP9](https://user-images.githubusercontent.com/45409515/157741215-5a730874-b619-4274-b624-14b129970279.png)

- sort


![PLP6](https://user-images.githubusercontent.com/45409515/157740876-6de8324f-64d3-4d3f-bb64-182f1d3c69d2.png)

![PLP4](https://user-images.githubusercontent.com/45409515/157740882-8ac435be-a147-4f49-86e3-6904d47b3a5e.png)

![PLP7](https://user-images.githubusercontent.com/45409515/157740873-a3b40414-6dc8-4e6f-b9b2-d95b03aff3cd.png)

- display grid/list view

![PLP2](https://user-images.githubusercontent.com/45409515/157740252-2aaec910-a30b-4f35-88e3-a40ed7df92e6.png)

![PLP3](https://user-images.githubusercontent.com/45409515/157740249-1633ed4a-0b27-4b62-bef2-9e4d05aa75b1.png)

![PLP1](https://user-images.githubusercontent.com/45409515/157740253-9cf2e3dd-fe5f-4c6b-91d0-b50272d832bf.png)
